examples/snap-plugin-collector-rand-streaming/main.go

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 package main
 
 import (
-	"github.com/intelsdi-x/snap-plugin-lib-go/examples/streaming/rand"
+	"github.com/intelsdi-x/snap-plugin-lib-go/examples/snap-plugin-collector-rand-streaming/rand"
```

Comment:
@jcooklin, I made PR with fix to your repo: https://github.com/jcooklin/snap-plugin-lib-go/pull/10